src/pages/community.html

```diff
@@ -167,10 +167,10 @@
 			<div class="footer">
 				<div class="footer__content">
 					<div class="footer__item">&copy; 2019 SAP SE. Made available under <nobr>Apache License 2.0</nobr></div>
-					<div class="footer__item"><a href="https://www.sap.com/about/legal/impressum.html" rel="noopener" target="_blank">Legal Disclosure</a></div>
+					<div class="footer__item"><a href="https://www.sap.com/about/legal/impressum.html" rel="noopener" target="_blank" class="link">Legal Disclosure</a></div>
```

Comment:
please remove unused CSS:
https://github.com/SAP/openui5-website/blob/master/src/components/footer/styles.css#L21:L29

Reply:
which part is unused?

Reply:
is it marked by the link above? Lines 21-29:
![screenshot 2019-02-14 at 12 33 57](https://user-images.githubusercontent.com/748043/52784271-d296e000-3054-11e9-87b8-5094cbcb40d0.png)